models/DataObject/ClassDefinition/Data/Wysiwyg.php

```diff
@@ -343,6 +343,7 @@ public function preGetData(/** mixed */ $container, /** array */ $params = []) /
         return Text::wysiwygText($data, [
                 'object' => $container,
                 'context' => $this,
+                'language' => $params['language'] ?? null,
```

Comment:
```suggestion
                'params' => $params ?? [],
```
Probably even better to pass all params, right?

Reply:
I am not sure. This params are directly passed to the link generator and i am not sure if the other params are  useful in the generator at all. 

Example:
![image](https://user-images.githubusercontent.com/9052094/208640328-dd853308-aeba-4d46-b2fa-0793c3176065.png)

Reply:
Hmm, that's true, seems to be redundant information 👍 Thanks for clarifying! 😊